docs/editor/code-editor/code-editor.component.scss

```diff
@@ -0,0 +1,185 @@
+:host {
+  // backgrounds
+  --code-editor-selection-background: color-mix(
+    in srgb,
+    var(--selection-background) 5%,
+    var(--octonary-contrast)
+  );
+  --code-editor-focused-selection-background: color-mix(
+    in srgb,
+    var(--selection-background) 12%,
+    var(--octonary-contrast)
+  );
+
+  // text base colors
+  --code-editor-text-base-color: var(--primary-contrast);
+
+  // tooltips
+  --code-editor-tooltip-background: color-mix(
+    in srgb,
+    var(--bright-blue),
+    var(--page-background) 90%
+  );
+  --code-editor-tooltip-color: var(--primary-contrast);
+  --code-editor-tooltip-border: 1px solid
+    color-mix(in srgb, var(--bright-blue), var(--page-background) 70%);
+  --code-editor-tooltip-border-radius: 0.25rem;
+
+  // autocomplete
+  --code-editor-autocomplete-item-background: var(--senary-contrast);
+  --code-editor-autocomplete-item-color: var(--primary-contrast);
+
+  // cursor
+  --code-name: var(--primary-contrast);
+  --code-editor-cursor-color: var(--code-name);
+  --code-variable-name: var(--bright-blue);
+  --code-property-name: var(--code-name);
+  --code-definition-keyword: var(--electric-violet);
+
+  // comments
+  --code-comment: var(--electric-violet);
+  --code-line-comment: var(--symbolic-gray);
+  --code-block-comment: var(--symbolic-brown);
+  --code-doc-comment: var(--code-comment);
+
+  // keywords
+  --code-keyword: var(--electric-violet);
+  --code-modifier: var(--code-keyword);
+  --code-operator-keyword: var(--code-keyword);
+  --code-control-keyword: var(--code-keyword);
+  --code-module-keyword: var(--code-keyword);
+
+  // structural
+  --code-brace: var(--vivid-pink);
+
+  // Misc
+  --code-bool: var(--bright-blue);
+  --code-string: var(--orange-red);
+  --code-regexp: var(--orange-red);
+
+  --code-tags: var(--bright-blue);
+  --code-component: var(--primary-contrast);
+  --code-type-name: var(--vivid-pink);
+  --code-self: var(--orange-red);
+
+  position: relative;
+}
+
+.adev-code-editor-tabs {
```

Comment:
I don't think we should do this now, but at some point we're going to want to replace these "adev" prefixes with "docs"

Reply:
Yeah, agreed, there is a ton of that type of cleanup that needs to happen.